app/components/Settings/Scheduler/util.js

```diff
@@ -0,0 +1,14 @@
+import {DIALECTS} from '../../../constants/constants.js';
+
+export function mapDialect(dialect) {
```

Comment:
Duplicate code: [code-editor.jsx](https://github.com/jakedex/falcon-sql-client/blob/498420c042490870b6669819ddfb8a1047cd2627/app/components/Settings/Preview/code-editor.jsx#L191-L200) already defines this code.

Please, add the following to `app/constants/constants.js` (untested code):
```js
export function getHighlightMode(dialect) {
    if (!SQL_DIALECTS_USING_EDITOR.includes(dialect)) {
        return 'text/plain';
    }

    return {
            [DIALECTS.APACHE_SPARK]: 'text/x-sparksql',
            [DIALECTS.MYSQL]: 'text/x-mysql',
            [DIALECTS.SQLITE]: 'text/x-sqlite',
            [DIALECTS.MARIADB]: 'text/x-mariadb',
            [DIALECTS.ORACLE]: 'text/x-plsql',
            [DIALECTS.POSTGRES]: 'text/x-pgsql',
            [DIALECTS.REDSHIFT]: 'text/x-pgsql',
            [DIALECTS.MSSQL]: 'text/x-mssql'
    }[dialect] || 'text/x-sql';
}
```

And update the following files accordingly:
- [ ] `app/components/Settings/Preview/code-editor.jsx`
- [ ] `app/components/Settings/Scheduler/createModal.jsx`
- [ ] `app/components/Settings/Scheduler/updateModal.jsx`